include/oboe/AudioStream.h

```diff
@@ -151,18 +199,44 @@ class AudioStream : public AudioStreamBase {
      */
     virtual int32_t getFramesPerBurst() = 0;
 
+    /**
+     * Indicates whether the audio stream is playing.
+     *
+     * THIS MAY BE REMOVED IN FUTURE. https://github.com/google/oboe/issues/208
```

Comment:
We should not document this at all except for deprecation.

Reply:
Fixed